metrics_collector/disk_collector.go

```diff
@@ -1,8 +1,10 @@
 package metricscollector
 
 import (
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/sirupsen/logrus"
+	"strings"
```

Comment:
nit: consider [organize package imports](https://github.com/longhorn/longhorn/wiki/coding-convention#organize-package-imports).

Reply:
@c3y1huang let's improve this later.

Reply:
https://github.com/longhorn/longhorn-manager/pull/2230